deployments/apiv2/services/users/src/main.py

```diff
@@ -347,6 +347,23 @@ async def _update_failed_login_attempts(con, account_type: str, id: str, count:
     await con.execute(update_query, id)
 
 
+async def _update_password(con, pw, previous_passwords, update_query, query_params):
+    ph = PasswordHasher()
+    phash = ph.hash(pw)
+    # make sure new password does not match any previous passwords on file
+    for prev_pw in previous_passwords:
+        try:
+            ph.verify(prev_pw, pw)
+        except VerifyMismatchError:
+            # passwords don't match, nothing else to do
+            continue
+        else:
+            # passwords match, return msg indicating that this is the case
+            raise UnableToUpdateAccountError()
+
+    await con.execute(update_query, *[phash, *query_params])
```

Comment:
Minor note for future, I think `await con.execute(update_query, phash, *query_params)` should work fine here

Reply:
oh duh yeah, I actually meant to change that